stack/database.py

```diff
@@ -117,8 +122,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion="9.5.4",
+    MultiAZ=False,
```

Comment:
You shoud keep it multizone (failover, latency...) c.f.: https://aws.amazon.com/rds/details/multi-az/